test/test_site/index.md

```diff
@@ -6,6 +6,12 @@ title: Hello World
 
 <div class="website-content">
 
+{{education_icon}}
+
+{{finalized_value}}
+
+{{reference_level_4}}
+
```

Comment:
Should add a heading to inform what is this section trying to test.

How about `# Test Variables`?